### PR TITLE
fix: use provider's region if region is not provided through options

### LINKED ISSUE #7: region.split is not a function
```
Serverless plugin "@pretty-fun-therapy/serverless-plugin-tag-cloud-watch-logs" initialization errored: region.split is not a function
```

Need to use `getRegion` instead of `getRegionSourceValue`
https://github.com/serverless/serverless/blob/master/lib/plugins/aws/provider/awsProvider.js#L453

### PR DESCRIPTION
**Pull request by chris-feist**
Resolves #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ custom:
     TagName2: TagValue2
 
 plugins:
-  - @pretty-fun-therapy/serverless-plugin-tag-cloud-watch-logs
+  - '@pretty-fun-therapy/serverless-plugin-tag-cloud-watch-logs'
```
Comment (chris-feist) [inline README.md]: Previous update was invalid yaml format

**Comment by chris-feist** [comment]
@clementheliou I updated the PR title. Let me know if this doesn't resolve your guidelines issue

**Comment by clementheliou** [comment]
> @clementheliou I updated the PR title. Let me know if this doesn't resolve your guidelines issue

Could you please rename the commit the same way ?

**Comment by chris-feist** [comment]
@clementheliou done

**Comment by chris-feist** [comment]
@clementheliou Let me know when you publish. Thanks!

**Comment by clementheliou** [comment]
@chris-feist I can't see it for now [on npm](https://www.npmjs.com/package/@pretty-fun-therapy/serverless-plugin-tag-cloud-watch-logs) but the version 1.0.2 has been published and I'm able to use it on our own projects.

Could you please give it a try and let us know if it does not work ?

**Comment by chris-feist** [comment]
@clementheliou I was able to get the latest version (even if NPM isn't updated) and verified it works correctly. Thanks 👍

**Pull request merged**